### PR TITLE
#1370 Use cypress test RESTful APIs with services and nodes page

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1370 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/support/commands.js [hunk 1]
```diff
@@ -17,6 +17,18 @@
 import 'cypress-testing-library/add-commands';
 
 import * as utils from '../utils';
+import { fetchZookeepers } from '../../src/api/zookeeperApi';
+import { fetchBrokers } from '../../src/api/brokerApi';
+
+Cypress.Commands.add('fetchBrokers', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/commands.js]: Can we simplify this to something like:

```js
Cypress.Commands.add('fetchBrokers', () => fetchBrokers());
```

As we're just returning a promise here 🤔

File: ohara-manager/client/cypress/integration/API.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+describe('API', () => {
+  it('Zookeeper', () => {
+    cy.fetchZookeepers()
+      .its('data')
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/API.test.js]: Not sure why we need both `its` and `then` here. Isn't `its` already returns the data that we need?

Comment (wu87988622) [reply ohara-manager/client/cypress/integration/API.test.js]: I will choose one more.

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

File: ohara-manager/client/package.json [hunk 3]
```diff
@@ -16,7 +16,9 @@
     "test:report": "CI=true yarn test --reporters=default --reporters=jest-junit",
     "test:coverage": "yarn test --coverage",
     "e2e": "cypress open",
-    "e2e:run": "cypress run"
+    "e2e:run": "cypress run",
+    "e2e:api": "cypress open --config integrationFolder=cypress/API",
```
Comment (konekoya) [inline ohara-manager/client/package.json]: api

**Comment by wu87988622** [comment]
retry manager-it

**Pull request merged**